### ISSUE TITLE
Boilerplate app newly created by vue-cli fails to start

### ISSUE THREAD

**Issue by dkulchenko** [vuejs/vue-cli#3441]
### Version
3.4.0

### Environment info
```
Environment Info:

  System:
    OS: macOS 10.14.3
    CPU: (4) x64 Intel(R) Core(TM) m7-6Y75 CPU @ 1.20GHz
  Binaries:
    Node: 11.9.0 - ~/n/bin/node
    Yarn: 1.13.0 - /usr/local/bin/yarn
    npm: 6.7.0 - ~/n/bin/npm
  Browsers:
    Chrome: 71.0.3578.98
    Firefox: 63.0.3
    Safari: 12.0.3
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0-beta.2
    @vue/babel-plugin-transform-vue-jsx:  1.0.0-beta.2
    @vue/babel-preset-app:  3.4.0
    @vue/babel-preset-jsx:  1.0.0-beta.2
    @vue/babel-sugar-functional-vue:  1.0.0-beta.2
    @vue/babel-sugar-inject-h:  1.0.0-beta.2
    @vue/babel-sugar-v-model:  1.0.0-beta.2
    @vue/babel-sugar-v-on:  1.0.0-beta.2
    @vue/cli-overlay:  3.4.0
    @vue/cli-plugin-babel: ^3.4.0 => 3.4.0
    @vue/cli-plugin-eslint: ^3.4.0 => 3.4.0
    @vue/cli-service: ^3.4.0 => 3.4.0
    @vue/cli-shared-utils:  3.4.0
    @vue/component-compiler-utils:  2.5.2
    @vue/preload-webpack-plugin:  1.1.0
    @vue/web-component-wrapper:  1.2.0
    eslint-plugin-vue: ^5.0.0 => 5.1.0
    vue: ^2.5.22 => 2.6.4
    vue-eslint-parser:  2.0.3
    vue-hot-reload-api:  2.3.1
    vue-loader:  15.6.2
    vue-style-loader:  4.1.2
    vue-template-compiler: ^2.5.21 => 2.6.4
    vue-template-es2015-compiler:  1.8.2
  npmGlobalPackages:
    @vue/cli: 3.4.0

```

### Steps to reproduce
1. Run `vue create testapp`
2. Select default option
3. `cd testapp; yarn serve`

### What is expected?
The app starts.

### What is actually happening?
```
~/testapp master
❯ yarn serve
yarn run v1.13.0
$ vue-cli-service serve
 INFO  Starting development server...
 ERROR  Error: custom keyword definition is invalid: data.errors should be boolean
Error: custom keyword definition is invalid: data.errors should be boolean
    at Ajv.addKeyword (/Users/daniilk/testapp/node_modules/ajv/lib/keyword.js:65:13)
    at module.exports (/Users/daniilk/testapp/node_modules/ajv-errors/index.js:10:7)
    at Object.<anonymous> (/Users/daniilk/testapp/node_modules/webpack-dev-server/node_modules/schema-utils/src/validateOptions.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:734:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
    at Function.Module._load (internal/modules/cjs/loader.js:558:3)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (internal/modules/cjs/helpers.js:20:18)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

~/testapp master
❯
```

**Comment by dkulchenko** [comment]
```bash
npm install -g @vue/cli-init
vue init webpack testapp
cd testapp; npm run dev
```

seems to work just fine with the same version of vue.

**Comment by passsy** [comment]
I have the same error.

As a quick fix I disabled the `definitionSchema` check in `node_modules/ajv/lib/keyword.js`

```js
var definitionSchema = {
  //...
  properties: {
    $data: {type: 'boolean'},
    async: {type: 'boolean'},
    // errors: {type: 'boolean'} <-- remove this line, keep coding while waiting for fix
  }
};
```

**Comment by dkulchenko** [comment]
Looks to be an upstream issue in `ajv`: https://github.com/epoberezkin/ajv/issues/941

**Comment by velocity-360** [comment]
The issue on the package that causes the break is here: https://github.com/epoberezkin/ajv/issues/941

**Comment by matthew-gordon** [comment]
Is there a way around this currently?

**Comment by karudo** [comment]
this bug is already fixed: epoberezkin/ajv#941
remove `node_modules` and `package-lock.json` and do `npm install` then

**Comment by LinusBorg** [comment]
I'll close this as resolved.
